bookkeeper-server/src/test/java/org/apache/bookkeeper/replication/AutoRecoveryMainTest.java

```diff
@@ -98,19 +96,7 @@ public void testAutoRecoverySessionLoss() throws Exception {
          */
         ZKMetadataClientDriver zkMetadataClientDriver1 = startAutoRecoveryMain(main1);
         ZooKeeper zk1 = zkMetadataClientDriver1.getZk();
-
-        // Wait until auditor gets elected
-        for (int i = 0; i < 10; i++) {
-            try {
-                if (main1.auditorElector.getCurrentAuditor() != null) {
-                    break;
-                } else {
-                    Thread.sleep(1000);
-                }
-            } catch (IOException e) {
-                Thread.sleep(1000);
-            }
-        }
+        await().until(() -> main1.auditorElector.getAuditor() != null);
```

Comment:
Why it's different than before? looks like the `getCurrentAuditor()` is different than `getAuditor()`